### DISCUSSION TITLE
API Endpoints

### DISCUSSION THREAD

**Discussion by OptusGroup** [coollabsio/coolify#54] [Feature Requests]
Hi,

I love the project and think it would be great if there were API endpoints at some point so that if someone wanted to implement Coolify into their own admin dashboard they could.

Is this by any chance on the roadmap?

Thanks for your hard work!

**Comment by andrasbacsai** [comment]
Hey,

It's an excellent idea, but not on the roadmap yet. I will add it to the roadmap, but cannot tell any date of implementing it.

**Reply by OptusGroup** [reply]
Thanks for the reply.

Considering that there is already a dashboard, an API could be built off the XHR requests. I, unfortunately, haven't yet had the chance to spin up Coolify yet. Was also wondering if this would be scalable and whether you have thought about adding serverless functions in the future?
